golang/sqlboiler/templates/10_resolver.tpl

```diff
@@ -33,9 +54,9 @@ func (r *Resolver) All{{$modelName}}s(ctx context.Context, args struct {
 	if args.Search != nil {
 		mods = append(mods, args.Search.QueryMods()...)
 	}
-	slice, err := dbmodel.{{$modelName}}s(r.db, mods...).All()
+	slice, err := dbmodel.{{$modelNamePlural}}(r.db, mods...).All()
```

Comment:
Changed model/table name pluralisation based on the method/func being called above
https://github.com/volatiletech/sqlboiler/blob/05d901865aad7a4f0e93771ec51efc18ee920a05/templates/13_all.tpl#L9